### ISSUE TITLE
Focus indicator not visible during keyboard navigation when using NVDA

### ISSUE THREAD

**Issue by iamrafan** [nvaccess/nvda#10727]
### Steps to reproduce:
1) Open https://codepen.io/iamrafan/pen/RwNdpOK?editable=true
2) Turn on NVDA
3) Navigate to the "New item" button using keyboard and press enter key
4) Observe the focus indicator on the first ```menuitem``` "Email Message" is not visible and upon pressing down arrow, the focus on the second ```menuitem``` "Calender Event" is visible

![NVDA focus bug](https://user-images.githubusercontent.com/22068434/73312451-9618b180-41dd-11ea-8a69-d30642ab0a13.gif)

### Actual behavior:
The focus indicator on the first ```menuitem``` "Email Message" is not visible after activating the "New item" ```menu``` using a keyboard

### Expected behavior:
The focus indicator on the first ```menuitem``` "Email Message" should be visible.

#### NVDA installed/portable/running from source: 
Installed

#### NVDA version: 
2019.2.1

#### Windows version: 
Windows 10 - Version 1903, OS Build - 18362.476

#### Name and version of other software in use when reproducing the issue: 
Chrome -Version 79.0.3945.130

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Adriani90** [comment]
@iamrafan are you still having this issue in NVDA 2024.1 Beta or 2023.3.3?
Are using browse mode? or focus mode when reproducing this issue?

**Comment by Adriani90** [comment]
We don't have any update upon request from the initial author since many years. I am closing as abandoned. if you are still having this issue with NVDA 2024.2 or newer, please comment and we can reopen.
